crates/crates-io/lib.rs

```diff
@@ -57,6 +57,7 @@ pub struct NewCrate {
     pub repository: Option<String>,
     pub badges: BTreeMap<String, BTreeMap<String, String>>,
     pub links: Option<String>,
+    pub rust_version: Option<String>,
```

Comment:
Forgot to call out that we need to update the publish API documentation to mention this and to remove the blurb in the Index documentation saying that they diverge that was added in #12040

Reply:
I can do this.

Reply:
Added to this PR. I did end up leaving a note regarding crates.io's behavior on the field, since I feel like it should be documented somewhere.